src/partials/install_instructions.html

```diff
@@ -1,131 +1,94 @@
+              src="/images/apple-line-gray.svg"
+            />
+            <span>Mac Silicon</span>
+          </button>
+          <!-- Windows -->
+          <button
+            class="mt-10 button px-5 py-2 button-outlined-gray whitespace-nowrap"
+            href=""
+            target="_blank"
+            rel="noopener"
           >
-            <img src="/images/docker.svg" alt="" width="28" height="21" />
-            <span class="text-sm font-medium">Install Docker</span>
-          </a>
+            <img src="/images/windows-line-gray.svg" />
+            <span>Windows</span>
+          </button>
         </div>
       </div>
-      <!-- Step -->
-      <div class="flex flex-col space-y-2">
-        <div class="text-gray-900 text-sm font-semibold">Step 2</div>
-        <!-- prettier-ignore -->
-        {{> code
-              id="docker-run-livebook"
-              code="docker run -p 8080:8080 -p 8081:8081 --pull always livebook/livebook"
-              isShell=true
-            }}
-      </div>
     </div>
+  </div>
+</div>
 
-    <!-- * Elixir instructions -->
+<!-- docker -->
+<div class="m-auto max-w-7x p-4 md:max-w-[932px]">
+  <div class="md:mt-12 mt-10 md:mb-28 mb-12 m-auto max-w-7xl">
     <div
-      id="install-elixir"
-      role="tabpanel"
-      aria-labelledby="link-install-elixir"
-      data-el="tab-content"
-      class="mt-6 flex flex-col space-y-6"
+      class="flex items-center max-w-[630px] rounded-lg border-gray-300 border-2 bg-gray-100 text-gray-700 p-3"
     >
-      <!-- Step -->
-      <div class="flex flex-col items-start space-y-2">
-        <div class="text-gray-900 text-sm font-semibold">Step 1</div>
-        <div>
-          <a
-            class="button button-gray"
-            href="https://elixir-lang.org/install.html"
-          >
-            <img src="/images/elixir.svg" alt="" width="12" height="21" />
-            <span class="text-sm font-medium">Install Elixir v1.13+</span>
-          </a>
-        </div>
-      </div>
-      <!-- Step -->
-      <div class="flex flex-col space-y-2">
-        <div class="text-gray-900 text-sm font-semibold">Step 2</div>
-        <!-- prettier-ignore -->
-        {{> code
-              id="elixir-install-local"
-              code="mix local.rebar --force && mix local.hex --force"
-              isShell=true
-            }}
-            {{> code
-              id="elixir-install-livebook"
-              code="mix escript.install hex livebook"
-              isShell=true
-            }}
-            {{> code
-              id="elixir-run-livebook"
-              code="livebook server"
-              isShell=true
-            }}
-      </div>
+      <img src="/images/icon-information.png" />
+      <p class="pl-5 text-base">
+        To run on Docker, embedded devices, or Elixir’s Mix.
+        <a
+          href="https://fly.io/launch/livebook"
+          class="hover:text-[#e44c75] underline"
+          >Check Read Me</a
+        >
```

Comment:
In other places we use border bottom instead of underling, this way the line doesn't cross letters such as `p` or `y` :)

```suggestion
        To run on Docker, embedded devices, or Elixir’s Mix
        <a
          href="https://github.com/livebook-dev/livebook#installation"
          class="border-b border-gray-700"
          >check our GitHub</a
        >.
```